cogeo_mosaic/backends/utils.py

```diff
@@ -1,26 +1,31 @@
+"""cogeo-mosaic.backends utility functions."""
+
+from typing import Any, Dict, List, Tuple
+
 import hashlib
 import itertools
 import json
 import os
 import zlib
-from typing import Any, Dict, List, Tuple
 
 import mercantile
 
 
 def find_quadkeys(mercator_tile: mercantile.Tile, quadkey_zoom: int) -> List[str]:
-    """Find quadkeys at desired zoom for tile
+    """
+    Find quadkeys at desired zoom for tile
 
     Attributes
     ----------
-    mercator_tile: mercantile.Tile
-        Input tile to use when searching for quadkeys
-    quadkey_zoom: int
-        Zoom level
+        mercator_tile: mercantile.Tile
+            Input tile to use when searching for quadkeys
+        quadkey_zoom: int
+            Zoom level
```

Comment:
I don't think the extra indentation is standard, within the package, e.g.:
https://github.com/developmentseed/cogeo-mosaic/blob/60749264bd8375c482ca5420ccefd39964edb3ce/cogeo_mosaic/utils.py#L87-L97

Reply:
yeah, to be honest I'm not sure, I found that adding this extra indentation enable pydoc preview within VSCode, So I'm trying to add this every time I have to touch a function. (e. g I updated most of rio-tiler code like this)